src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsPullsListener.php

```diff
@@ -322,38 +330,7 @@ protected function checkPullLabels($hookData, Github $github, Logger $logger, $p
 		}
 
 		// Get the files modified by the pull request
-		try
-		{
-			$files = $github->pulls->getFiles($project->gh_user, $project->gh_project, $hookData->pull_request->number);
-		}
-		catch (InvalidResponseCodeException $e)
-		{
-			$logger->error(
-				sprintf(
-					'Error retrieving modified files for GitHub item %s/%s #%d',
-					$project->gh_user,
-					$project->gh_project,
-					$hookData->pull_request->number
-				),
-				['exception' => $e]
-			);
-
-			$files = [];
-		}
-		catch (\DomainException $e)
-		{
-			$logger->error(
-				sprintf(
-					'Error retrieving modified files for GitHub item %s/%s #%d',
-					$project->gh_user,
-					$project->gh_project,
-					$hookData->pull_request->number
-				),
-				['exception' => $e]
-			);
-
-			$files = [];
-		}
+		$files = $this->getChangedFilesByPullRequest($hookData, $github, $logger, $project)
```

Comment:
fixed with the last commit: https://github.com/joomla/jissues/pull/770/commits/61aab5b6c0adb4ab57325301111e78ab54da9de5 Thanks